app/src/main/java/com/magdamiu/androidfundamentalsmai2021/MainActivity.java

```diff
@@ -54,7 +54,6 @@ public void displayGreetingsOnClick(View view) {
             textViewGreetings.setText(inputName);
         } else {
             editTextName.setError(getString(R.string.error_insert_name));
```

Comment:
It is a good practice to format the code with CTRL + ALT + L before you commit it and to remove the unnecessary spaces